arpack-ng-config.cmake.in

```diff
@@ -18,24 +17,9 @@ set(exec_prefix "@exec_prefix@")
 set(libdir "@libdir@")
 set(includedir "@includedir@")
 
-set(arpack_ng_INCLUDE_DIRS "${includedir}/arpack;${MPI_Fortran_INCLUDE_DIRS};${MPI_C_INCLUDE_DIRS};${MPI_CXX_INCLUDE_DIRS}")
-
-if(EXISTS "${libdir}/libarpack.a")
-  set(arpack_ng_LIBRARIES "${libdir}/libarpack.a")
-elseif(EXISTS "${libdir}/libarpack.so")
-  set(arpack_ng_LIBRARIES "${libdir}/libarpack.so")
-endif()
-
-if(EXISTS "${libdir}/libparpack.a")
-  set(parpack_ng_LIBRARIES "${libdir}/libparpack.a;${arpack_ng_LIBRARIES};${MPI_Fortran_LIBRARIES};${MPI_C_LIBRARIES};${MPI_CXX_LIBRARIES}")
-elseif(EXISTS "${libdir}/libparpack.so")
-  set(parpack_ng_LIBRARIES "${libdir}/libparpack.so;${arpack_ng_LIBRARIES};${MPI_Fortran_LIBRARIES};${MPI_C_LIBRARIES};${MPI_CXX_LIBRARIES}")
-endif()
-
 # Create arpack targets.
 add_library(ARPACK::ARPACK INTERFACE IMPORTED)
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${arpack_ng_INCLUDE_DIRS}")
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "${arpack_ng_LIBRARIES}")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack")
```

Comment:
Not sure to know if this will work: @turboencabulator , did you test ? https://github.com/opencollab/arpack-ng/pull/259#issuecomment-603829535

Reply:
I copied one of the tests and used the example CMake snippet in the comments, and it worked.

From what I've read about CMake, it's discouraged to hard-code paths into these kinds of files.  So it's better in this regard, but maybe not ideal.

Reply:
I agree. Most important is you tested it OK.